src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -585,6 +592,15 @@ public Builder appCommandOutputFormat(String appCommandOutputFormat) {
     }
 
     /**
+     * Sets structured json logs for the stderr output. Supported values include 'never' (default),
+     * 'always', 'terminal', etc.
+     */
+    public Builder appCommandShowStructuredLogs(String appCommandShowStructuredLogs) {
```

Comment:
actually it looks @loosebazooka has some code in the experimental package that addresses all of this.